Package.swift

```diff
@@ -6,9 +6,9 @@ let package = Package(
     name: "swift-graphql",
     platforms: [
         .iOS(.v15),
-        .macOS(.v10_15),
-        .tvOS(.v13),
-        .watchOS(.v6)
+        .macOS(.v12),
+        .tvOS(.v15),
+        .watchOS(.v8)
```

Comment:
Can you explain why this is necessary? While we do have this planned for the next major release, that is not intended for 5.x since it should be considered a breaking change.

Reply:
the session task delegate is available from iOS v15, maxOS v12 see
https://developer.apple.com/documentation/foundation/urlsessiontask/3746977-delegate